src/util/audio.hpp

```diff
@@ -0,0 +1,122 @@
+#pragma once
+#include <Windows.h>
+#include <filesystem>
+#include <mmsystem.h>
+
+#pragma comment(lib, "winmm.lib")
+
+namespace big::audio
+{
+	enum string_code
```

Comment:
This belongs in either gta/enums.hpp or core/enums.hpp, it depends if the game uses them or not, I have no idea where they come from.

Reply:
Core

Reply:
and I made them